lms/templates/footer.html

```diff
@@ -25,6 +25,21 @@
             </ul>
           </nav>
 
+          <div class="wrapper-logo">
+            <p>
+              <a href="/">
+                ## The default logo is a placeholder.
+                ## You can either replace this link entirely or update
+                ## the FOOTER_ORGANIZATION_IMAGE in Django settings.
+                ## If you customize FOOTER_ORGANIZATION_IMAGE, then the image
+                ## can be included in the footer on other sites
+                ## (e.g. a blog or marketing front-end) to provide a consistent
+                ## user experience.  See the branding app for details.
+                <img alt="organization logo" src="${footer['logo_image']}">
```

Comment:
This is a straight up copy of what's in the "else" block of the conditional...

Reply:
I'm not familiar with edx-platform, but should this alt text be localized?

Reply:
Good catch, @abutterworth, I just copied over the html stanza from the `else` block. I'll fix both instances.